hooks/honeybadger/honeybadger.go

```diff
@@ -0,0 +1,36 @@
+package logrus_honeybadger
+
+import (
+	"github.com/DavidHuie/gobadger"
+	"github.com/Sirupsen/logrus"
+)
+
+// HoneybadgerHook to send exceptions to an exception-tracking service compatible
+// with the Honeybadger API.
+type HoneybadgerHook struct {
+	conn *gobadger.Conn
+}
+
+func NewHoneyBadgerHook(apiKey string) *HoneybadgerHook {
+	return &HoneybadgerHook{gobadger.NewConn(apiKey)}
+}
+
+func (hook *HoneybadgerHook) Fire(entry *logrus.Entry) error {
+	err := hook.conn.Error(entry.Message)
```

Comment:
Excellent point on the Airbrake hook, we should make a timeout for that one as well!

I'm cool with a timeout yep, it's not ideal behaviour (I'd add something like a circuit break for that, if your software is really critical I'd take a look at that, I'm happy to discuss it more too if you like), but enough to ship it for sure.